test/e2e/suite/command/cp.go

```diff
@@ -56,6 +56,10 @@ var _ = Describe("ORAS beginners:", func() {
 			ORAS("cp", RegistryRef(ZOTHost, ImageRepo, foobar.Tag)).ExpectFailure().MatchErrKeyWords("Error:").Exec()
 		})
 
+		It("should fail when no tag or digest is provided for source target", func() {
```

Comment:
Good catch. Still need tests for `blob delete` and `blob fetch` to make sure `tag` doesn't be presented in the output.

Reply:
Added e2e tests for `blob delete` and `blob fetch`